### PR TITLE
Update syntax / function calls for logistic reg

### PR DESCRIPTION
**Pull request by jeremiahpslewis**
Fixes a few issues that prevented the notebook from running (and one small syntax cleanup).

Note: I have an M1 Mac and can't at the moment actually spin up jupyter notebooks (embarrassing, admittedly), this means I haven't been able to re-run the notebook for updated cell outputs.

### CODE REVIEW

File: 2_LogisticRegression.ipynb [hunk 1]
```diff
@@ -19190,10 +19171,10 @@
...
     "function prediction(x::Matrix, chain, threshold)\n",
     "    # Pull the means from each parameter's sampled values in the chain.\n",
-    "    intercept = mean(chain[:intercept].value)\n",
-    "    student = mean(chain[:student].value)\n",
-    "    balance = mean(chain[:balance].value)\n",
-    "    income = mean(chain[:income].value)\n",
+    "    intercept = mean(chain[:intercept].data)\n",
+    "    student = mean(chain[:student].data)\n",
+    "    balance = mean(chain[:balance].data)\n",
+    "    income = mean(chain[:income].data)\n",
```
Comment (cpfiffer) [inline 2_LogisticRegression.ipynb:19177]: These should probably be 

```suggestion
    "    intercept = mean(Array(chain[:intercept]))\n",
    "    student = mean(Array(chain[:student]))\n",
    "    balance = mean(Array(chain[:balance]))\n",
    "    income = mean(Array(chain[:income]))\n",
```

**Review by cpfiffer — COMMENTED** [review]
Just one minor comment, other than that, thank you!

**Comment by jeremiahpslewis** [comment]
Thanks for the PR! Can you explain your comment? Is there a computational difference or is this just more 'idiomatic' given multiple dispatch?

**Comment by jeremiahpslewis** [comment]
@cpfiffer resolved :)

**Review by cpfiffer — APPROVED** [review]
Excellent, thank you!

**Pull request merged**